src/pgduckdb_filter.cpp

```diff
@@ -1,20 +1,21 @@
+#include "pgduckdb/pgduckdb_filter.hpp"
+
 #include "duckdb.hpp"
 #include "duckdb/planner/filter/constant_filter.hpp"
 #include "pgduckdb/pgduckdb_types.hpp"
+#include "pgduckdb/pg/declarations.hpp"
+#include "pgduckdb/pg/datum.hpp"
+#include "pgduckdb/pgduckdb_detoast.hpp"
+
 
 extern "C" {
-#include "postgres.h"
-#include "catalog/pg_type.h"
-#include "utils/builtins.h"
-#include "utils/date.h"
-#include "utils/timestamp.h"
-#if PG_VERSION_NUM >= 160000
-#include "varatt.h"
-#endif
+// Exceptional include of PG Header which doesn't contain any function declaration
+// only macro definitions and types.
+#include "catalog/pg_type_d.h"
 }
 
-#include "pgduckdb/pgduckdb_filter.hpp"
```

Comment:
I thought there were filters we can't push down?

Reply:
Yeah, but those we then also shouldn't receive from DuckDB. They should simply be executed by DuckDB.

Reply:
Ah yeah because everything will be emitted to DuckDB values after 477 lands. I think you're right. I'll keep this one on standby but will move to another file.